### DISCUSSION TITLE
How to add text after price in Cart Line Items?

### DISCUSSION THREAD

**Discussion by gswebs** [woocommerce/woocommerce#48981] [WooCommerce, Blocks, and Site Editing]
I am trying to add the text after price and before sale badge in the Cart Line Items. 

Is there any way or filter to do that? 

I have tried the following code, but it does not work:

          const { registerCheckoutFilters } = window.wc.blocksCheckout;
            
            const modifySubtotalPriceFormat = (defaultValue, extensions, args, validation ) => {
                
                //cosole.log(args);
                
	            const isCartContext = args?.context === 'cart';
            
	            if ( ! isCartContext ) {
		            return defaultValue;
	            }
            
	            return '<price/> <div className="custom-txt">per item</div>';
            };
            
            registerCheckoutFilters( 'example-extension', {
	            subtotalPriceFormat: modifySubtotalPriceFormat,
            } ); 

![image](https://github.com/woocommerce/woocommerce/assets/51439927/5f16da00-46fe-4220-9deb-6f137e07ec26)

**Comment by opr** [comment]
@gswebs unfortunately there's no way to add it _underneath_ the price, but that filter would allow you to add it directly after the price.

<img width="586" alt="image" src="https://github.com/user-attachments/assets/de1cac28-099c-4c18-a4c1-4a5ec10029ef">

```ts
const { registerCheckoutFilters } = window.wc.blocksCheckout;

const modifySubtotalPriceFormat = (
	defaultValue,
	extensions,
	args,
	validation
) => {
	const isCartContext = args?.context === 'cart';

	if ( ! isCartContext ) {
		return defaultValue;
	}

	return '<price/> per item.';
};

registerCheckoutFilters( 'example-extension', {
	subtotalPriceFormat: modifySubtotalPriceFormat,
} );
```
